src/main/java/io/DukeNUSPrinter.java

```diff
@@ -0,0 +1,117 @@
+package io;
+
+import tasks.Task;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+/**
+ * A tool to print messages to the terminal. Used by Duke chatbot.
+ */
+public class DukeNUSPrinter {
+    public static final String LOGO = ""
+            + "     ____        _          __    _ _   _ ______\n"
+            + "    |  _ \\ _   _| | _____  |   \\ | | | | | _____|\n"
+            + "    | | | | | | | |/ / _ \\ | |\\ \\| | | | |_____ |\n"
+            + "    | |_| | |_| |   <  __/ | | \\ \\ | |_| |____| | \n"
+            + "    |____/ \\__,_|_|\\_\\___| |_|  \\__|_____|______|\n";
+    public static final String INDENT = "    ";
+
+    /**
+     * Prints a horizontal line made of underscores to the terminal.
+     */
+    public static void printHorizontalLine() {
+        System.out.println(INDENT + "____________________________________________________________");
+    }
+
+    /**
+     * Prints a DukeNUS message bound by horizontal lines above and below to the console. Only single lines should be used.
+     *
+     * @param message The single line message to be bound by horizontal lines
+     */
+    public static void printMessage(String message) {
+        printHorizontalLine();
+        System.out.println(INDENT + message);
+        printHorizontalLine();
+    }
+
+    /**
+     * Prints a message to the console whenever a task is added to the tasks array.
+     *
+     * @param taskDisplayDescription The formatted description of the task with indications of its isDone and Task object type.
+     */
+    public static void printAddedTask(String taskDisplayDescription, int totalTaskCount) {
+        printHorizontalLine();
+        System.out.println(INDENT + "Got it. I've added this task:");
+        System.out.println(INDENT + taskDisplayDescription);
```

Comment:
Since you have the printMessage() method, a similar method could be used instead of using system.out.println.